model/src/main/java/uk/gov/hmcts/payment/api/service/FakeAccountServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.payment.api.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import uk.gov.hmcts.payment.api.dto.AccountDto;
+import uk.gov.hmcts.payment.api.util.AccountStatus;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Profile("mock-liberata")
+@Primary
+@Service
+public class FakeAccountServiceImpl implements AccountService<AccountDto, String> {
```

Comment:
you can use @MockBean in that controller test and define mock mappings there itself.

Reply:
This is for developer-runtime only, not for tests so not used in tests